scripts/shared/lib/utils

```diff
@@ -53,37 +53,33 @@ function with_context() {
 }
 
 # Run cluster commands in parallel.
-# 1st argument is the numbers of the clusters to run for, supports "1 2 3" or "{1..3}" for range
+# 1st argument is the cluster names for which to run.
 # 2nd argument is the command to execute, which will have the $cluster variable set.
 # 3rd argument and so forth get passed to the command.
 function run_parallel() {
-    local clusters cmnd
-    clusters=$(eval echo "$1")
-    cmnd=$2
+    local cmnd=$2
     declare -A pids
-    for i in ${clusters}; do
+    for cluster in $(eval echo "$1"); do
```

Comment:
```suggestion
    for cluster in $1; do
```

The `eval` was there to support `{1..3}` syntax but since it's not used anymore then no need for it